### ISSUE TITLE
The llm model is support infer use `vllm`  or use multi GPUs?

### ISSUE THREAD

**Issue by Ox0400** [FunAudioLLM/CosyVoice#228]
The llm module is support infer use `vllm`  or use multi GPUs?
If not, when will these features be implemented ?

**Comment by aluminumbox** [comment]
I have no knowledge in vllm, our repo only provide python inference code by now

**Comment by Ox0400** [comment]
> I have no knowledge in vllm, our repo only provide python inference code by now

The vllm is `A high-throughput and memory-efficient inference and serving engine for LLMs`

 https://github.com/vllm-project/vllm .


Question 2, is support infer use multi GPUs, Is there a plan to support multi GPUs?

**Comment by aluminumbox** [comment]
> > I have no knowledge in vllm, our repo only provide python inference code by now
> 
> The vllm is `A high-throughput and memory-efficient inference and serving engine for LLMs`
> 
> https://github.com/vllm-project/vllm .
> 
> Question 2, is support infer use multi GPUs, Is there a plan to support multi GPUs?

no, I don't think multi gpu for one service is a good idea. We will try to reduce rtf on a single GPU

**Comment by Ox0400** [comment]
> > > I have no knowledge in vllm, our repo only provide python inference code by now
> > 
> > 
> > The vllm is `A high-throughput and memory-efficient inference and serving engine for LLMs`
> > https://github.com/vllm-project/vllm .
> > Question 2, is support infer use multi GPUs, Is there a plan to support multi GPUs?
> 
> no, I don't think multi gpu for one service is a good idea. We will try to reduce rtf on a single GPU

Thank you for your reply.
